TSC-Charter.md

```diff
@@ -74,9 +74,25 @@ including:
 * Development process and any coding standards.
 * Mediating technical conflicts between Collaborators or Foundation
   projects.
+* Official communication about the above and what they govern, and the development process of said communication.
```

Comment:
"Official communication" is too vague here. I've no idea what would make it "official"